gnomad/resources/grch38/reference_data.py

```diff
@@ -138,7 +139,20 @@ def _import_clinvar(**kwargs) -> hl.Table:
                 "min_partitions": 400,
                 "reference_genome": "GRCh38",
             },
-        )
+        ),
+        "b154": TableResource(
+            path="gs://gnomad-public/resources/grch38/dbsnp/dbsnp_b154_grch38_all_20200514.ht",
+            import_func=import_sites_vcf,
+            import_args={
+                "path": "gs://gnomad-public/resources/grch38/dbsnp/dbsnp_b154_grch38_all_GCF_000001405.38_20200514.vcf.bgz",
+                "header_file": "gs://gnomad-public/resources/grch38/dbsnp/dbsnp_b154_grch38_all_GCF_000001405.38_20200514.vcf.header",
+                "force_bgz": True,
+                "contig_recoding": DBSNP_B154_CHR_CONTIG_RECODING,
+                "skip_invalid_loci": True,
+                "min_partitions": 400,
+                "reference_genome": "GRCh38",
+            },
+        ),
```

Comment:
In other repos, we've put the latest version at the top of the resource dictionary, closest to the default version. I think flipping this with b151 to match that set up would be good.